### ISSUE TITLE
Please Update Your Github tag to reflect your latest stable release version.

### ISSUE THREAD

**Issue by jumpsmm7** [AdguardTeam/AdGuardHome#4550]
Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [ X ] I am running the latest version
- [ X ] I checked the documentation and found no answer
- [ X ] I checked to make sure that this issue has not already been filed

### Issue Details

Please Update Your Github tag to reflect your latest stable release version.

**Comment by agneevX** [comment]
Has 107.6 been pulled back? Because 107.5 is marked as `Latest`, per GitHub. CC @ainar-g.

Otherwise, the Docker Hub tag stands corrected since 0.107.6 is the latest stable release.

**Comment by jumpsmm7** [comment]
> Has 107.6 been pulled back? Because 107.5 is marked as `Latest`, per GitHub. CC @ainar-g.
> 
> Otherwise, the Docker Hub tag stands corrected since 0.107.6 is the latest stable release.

I don't think it ever got updated with the inclusion of 0.107.6 is the latest stable release.
My package handler uses the binaries from github and checks against the github tag for version checks, so it reports the 107.5 as the latest, but I have 107.6 installed.

**Comment by jumpsmm7** [comment]
> 107.6 looks like the latest release now.

It appears you are correct.
